src/redux/categories/Categories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const STATUS = 'STATUS';
```

Comment:
- [ ] Similarly, assigning value to _CHECK_STATUS_ action type with respect to the the _duck patterns approach_ should be something like below:
```ruby
      CHECK_STATUS = bookstore/categories/CHECK_STATUS;
```
 
Kindly ensure to improve upon this aspect.

Reply:
Thanks for sharing @developertariq 👍